packages/api/cms-api/src/user-permissions/auth/user-permissions.guard.ts

```diff
@@ -57,7 +64,7 @@ export class UserPermissionsGuard implements CanActivate {
             // The first level consists of submitted scopes and affected entities
             // The only case that there is more than one scope in the second level is when the ScopedEntity returns more scopes
             return requiredPermissions.some((permission) =>
-                requiredContentScopes.every((contentScopes) =>
+                requiredContentScopes?.every((contentScopes) =>
```

Comment:
Adding the optional operator here isn't idea. This is because we're in a inner function here and TS thinks that it might have become undefined again. But I think it's okay. Alternatively I could throw an error here to make TS happy.

Reply:
Fixed by moving loading of the required content scopes back to the initial position.